apps/showcase/src/components/utilities/otter-icon/otter-icon-pres.component.ts

```diff
@@ -0,0 +1,32 @@
+import { ChangeDetectionStrategy, Component, computed, input, ViewEncapsulation } from '@angular/core';
+import { O3rComponent } from '@o3r/core';
+
+@O3rComponent({ componentType: 'Component' })
+@Component({
+  selector: 'o3r-otter-icon-pres',
+  standalone: true,
+  templateUrl: './otter-icon-pres.template.html',
+  styleUrls: ['./otter-icon-pres.style.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class OtterIconPresComponent {
```

Comment:
Why a component more that a pipe?
Here you just want to calculate the `realUrl` of an icon (I also think that the `alt` should not be the url of the icon but a meaningful short description)

Reply:
I agree in general that the `alt` should be a meaningful description, but in that specific case we are displaying the image that has been uploaded to create a pet. There is no description for that image and I thought in that specific case the url was relevant
As for component vs pipe, I used a component with the idea that we could want to change some html/css on these images but at the end it might not be the case, so I'm ok to only use a pipe